### ISSUE TITLE
No IEV standards fetching

### ISSUE THREAD

**Issue by opoudjis** [relaton/relaton-iec#51]
````
bundle exec relaton fetch "IEC 60050"
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
[relaton-iec] ("IEC 60050") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050. The code must be exactly like it is on the standards website.
[relaton-iec] (There was no match for , though there were matches found for 2007, 2009, 2010, 1998, 2013, 2020, 2014, 2016, 2017, 1988, 1992, 1993, 1995, 1996, 1991, 1997, 1994, 1982.)
[relaton-iec] If you wanted to cite all document parts for the reference, use "IEC 60050 (all parts)".
If the document is not a standard, use its document type abbreviation (TS, TR, PAS, Guide).





bundle exec relaton fetch "IEC 60050-102"
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
[relaton] (IEC 60050-102) not found.
No matching bibliographic entry found





bundle exec relaton fetch "IEC 60050-102:2007"
Calling `DidYouMean::SPELL_CHECKERS.merge!(error_name => spell_checker)' has been deprecated. Please call `DidYouMean.correct_error(error_name, spell_checker)' instead.
[relaton-iec] ("IEC 60050-102:2007") fetching...
[relaton-iec] WARNING: no match found online for IEC 60050-102:2007:2007. The code must be exactly like it is on the standards website.
[relaton-iec] The provided document part may not exist, or the document may no longer be published in parts.
No matching bibliographic entry found
````

**Comment by opoudjis** [comment]
Until this is fixed, I need to disable metanorma-standoc, cleanup_spec.rb, "separates IEV citations by top-level clause"

**Comment by opoudjis** [comment]
> It seems the issue with index. We use an index to search filenames. For IEV we use API to fetch last updates only, so the index should be updated partly. I see the documents in the repository but in the index they are missed. Looks like the index isn't updated correctly. I'm going to recreate the index entirely. Sorry, but I'm unable to rewrite the code today. I've started full repository update, it should recreate the index from scratch. After the repository update you can remove your local ~/.relaton/iec/index.yaml and try to fetch the documents again

**Comment by andrew2net** [comment]
fixed in v 1.14.3
